scripts/testing/dockerconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"auths": {
+		"127.0.0.1:15000": {
+      "auth": "dGVzdHVzZXI6dGVzdHB3ZA=="
+    }
```

Comment:
Indentation is a bit weird :)

Also is the `HttpHeaders` and `credStore` important? Reading it could suggest that it's used to test the `osx` patch you've done but I don't think it's the case (as `auth` is not empty).

Reply:
Yes, that's true. I fixed the indentation and removed `HttpHeaders` and `credStore`  because it only confuses.  Thanks for pointing this out.